docs/theia-cli.md

```diff
@@ -36,14 +36,20 @@ theia help
 
 ## Usage
 
-To see the list of available commands and options, run `theia help`. Currently,
-we have 5 commands for the NetworkPolicy Recommendation feature:
+To see the list of available commands and options, run `theia help`.
+We currently have below commands for NetworkPolicy Recommendation and ClickHouse:
+
+NetworkPolicy Recommendation:
 
 - `theia policy-recommendation run`
 - `theia policy-recommendation status`
 - `theia policy-recommendation retrieve`
 - `theia policy-recommendation list`
 - `theia policy-recommendation delete`
 
+ClickHouse:
+
+- `theia clickhouse status`
+
```

Comment:
As NetworkPolicy Recommendation has a separate file to describe how each commands work, the description of NetworkPolicy Recommendation commands here is brief. But for the ClickHouse commands, I think you still need to introduce the command functionality in detail here, like explaining how flags work and displaying some sample outputs.

Reply:
Thanks for the suggestion. I've updated the doc. Please take a look to see if it's clear enough. Thanks!